### PR TITLE
fix: Set watch to true

### LINKED ISSUE #69: Does not recompile the crate after encountering an error
### Steps to reproduce:

* Edit `crate/src/lib.rs` to have a syntax error
* `npm run start` -- note the compilation errors
* Remove the syntax error

### Expected results

The crate is automatically rebuilt with `wasm-pack build` after the syntax error is removed and the file is saved.

### Actual results

The crate is not rebuilt.

### PR DESCRIPTION
**Pull request by drager**
This will fix #69 once https://github.com/wasm-tool/wasm-pack-plugin/pull/43 lands.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ const dist = path.resolve(__dirname, "dist");
 const WasmPackPlugin = require("@wasm-tool/wasm-pack-plugin");
 
 module.exports = {
+  watch: true,
```
Comment (sendilkumarn) [inline webpack.config.js]: This is default in `webpack-dev-server` so I hope we will not need this

Comment (drager) [reply webpack.config.js]: Right, I thought so to, but when I logged the `compiler.options.watch` in https://github.com/wasm-tool/wasm-pack-plugin I got `undefined` if I didn't set it to true.

Comment (sendilkumarn) [reply webpack.config.js]: I think it should run the other way around. And for sure webpack-dev-server adds this by default. I think I found the issue. I will raise a PR for `wasm-pack-plugin`

Comment (drager) [reply webpack.config.js]: Alright, great. Then we can close this PR once that's merged.

**Comment by sendilkumarn** [comment]
@drager So sorry that I have to close the PR

**Pull request closed without merging**